browser/resources/settings/brave_content_page/containers.ts

```diff
@@ -83,7 +89,12 @@ export class SettingsBraveContentContainersElement extends SettingsBraveContentC
   }
 
   onAddContainerClick_() {
-    this.editingContainer_ = { id: '', name: '' }
+    this.editingContainer_ = {
+      id: '',
+      name: '',
+      icon: Icon.kPersonal,
```

Comment:
maybe we can add a `kDefault = kPersonal` in mojom and use it in these scenarios? can you try this please? maybe put it last in the enum after all elements:
```
enum Icon {
  ...
  kMessaging = 10,

  kDefault = kPersonal,
};
```

also can be usable as a fallback when something goes wrong.